pkg/scheduler/framework/preemption/preemption.go

```diff
@@ -122,6 +123,8 @@ type Interface interface {
 	// Note that both `state` and `nodeInfo` are deep copied.
 	SelectVictimsOnNode(ctx context.Context, state *framework.CycleState,
 		pod *v1.Pod, nodeInfo *framework.NodeInfo, pdbs []*policy.PodDisruptionBudget) ([]*v1.Pod, int, *framework.Status)
+	// DeletePod deletes the given victim pod from API server for preemption.
+	DeletePod(ctx context.Context, cs kubernetes.Interface, victim, preemptor *v1.Pod) error
```

Comment:
> all those member methods might have different behavior compared with the default preemption

From the comments, I have no strong preference to add this method, its' a common one.

At the least, I think it should be named like `DeleteVictim`, sounds better? and maybe `metav1.DeleteOptions{}` can also be pass-in for extendibility?

Ask for tech advices from @Huang-Wei

Reply:
Changed the name to DeleteVictim. The DeleteOptions field is in control of the plugin owner.

Reply:
exactly